extensions/amp-live-list/0.1/amp-live-list.js

```diff
@@ -206,9 +206,11 @@ export class AmpLiveList extends AMP.BaseElement {
 
     const maxItems = this.element.getAttribute('data-max-items-per-page');
     user().assert(Number(maxItems) > 0,
-        `amp-live-list#${this.liveListId_} must have ` +
-        'data-max-items-per-page attribute with numeric value. ' +
-        `Found ${maxItems}`);
+        'amp-live-list # %s must have data-max-items-per-page attribute with'
```

Comment:
did you intend to add spaces around `#`, they are added here, but below, you have `amp-live-list# %s`. Should just make them consistent. I think without the space is nice because then it is a valid css selector for the element.

Reply:
Done